shapely/coords.py

```diff
@@ -47,6 +75,27 @@ def __getitem__(self, key):
             raise TypeError("key must be an index or slice")
 
     def __array__(self, dtype=None, copy=None):
+        """Return a copy of the coordinate array.
+
+        Parameters
+        ----------
+        dtype : data-type, optional
+            The desired data-type for the array.
+        copy : bool, optional
+            If True (default), a copy of the array is returned. If False, the
+            array is returned as-is.
```

Comment:
This is recent from #2053 and has a default `copy=None`, so the docstring would need adjusting. Further help from [numpy](https://numpy.org/devdocs/numpy_2_0_migration_guide.html#adapting-to-changes-in-the-copy-keyword). Maybe it should be something like this:
```suggestion
        copy : bool, optional
            Using None (default) or True, a copy of the array is always returned.
```

Reply:
OK. I also added that `copy=False` will raise a ValueError.